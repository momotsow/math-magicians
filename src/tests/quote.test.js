import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Quote from '../components/quote';

describe('Testing quote page to render as expected', () => {
  const quotePage = render(
    <Router>
      <Quote />
    </Router>,
  );
  it('Quote should render correctly', () => {
    // Assert
    expect(quotePage).toMatchSnapshot();
  });
});
