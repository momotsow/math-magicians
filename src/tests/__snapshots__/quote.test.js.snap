// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing quote page to render as expected Quote should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="homeContainer"
      >
        <div
          class="content"
        >
          <div
            class="contentTitle"
          >
            <h1>
              Quotes
            </h1>
          </div>
          <div
            class="contentDescription"
          >
            <p>
              Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul thurston.
              <br />
              <br />
            </p>
          </div>
        </div>
        <div
          class="footer"
        >
          <p>
            Developed by
            <a
              href="https://github.com/momotsow/math-magicians"
            >
               KP Mkansi
            </a>
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="homeContainer"
    >
      <div
        class="content"
      >
        <div
          class="contentTitle"
        >
          <h1>
            Quotes
          </h1>
        </div>
        <div
          class="contentDescription"
        >
          <p>
            Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul thurston.
            <br />
            <br />
          </p>
        </div>
      </div>
      <div
        class="footer"
      >
        <p>
          Developed by
          <a
            href="https://github.com/momotsow/math-magicians"
          >
             KP Mkansi
          </a>
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
